sal_interfaces/Rotator/Rotator_Commands.xml

```diff
@@ -164,73 +97,6 @@
     </item>
 </SALCommand>
 
-<SALCommand>
-    <Subsystem>Rotator</Subsystem>
-    <Version>3.5.1</Version>
-    <Author></Author>
-    <EFDB_Topic>Rotator_command_positionSet</EFDB_Topic>
-    <Alias>positionSet</Alias>
-    <Device></Device>
-    <Property></Property>
-    <Action></Action>
-    <Value></Value>
-    <Description>Set the desired fixed position of the rotator. Issue the move command to actually move the rotator.</Description>
-    <item>
-        <EFDB_Name>angle</EFDB_Name>
-        <Description>angle</Description>
-        <IDL_Type>double</IDL_Type>
-        <Units>deg</Units>
-        <Count>1</Count>
-    </item>
-</SALCommand>
-
-<SALCommand>
-    <Subsystem>Rotator</Subsystem>
-    <Version>3.5.1</Version>
-    <Author></Author>
-    <EFDB_Topic>Rotator_command_moveConstantVelocity</EFDB_Topic>
-    <Alias>moveConstantVelocity</Alias>
-    <Device></Device>
-    <Property></Property>
-    <Action></Action>
-    <Value></Value>
-    <Description>Move rotator at the velocity and for the duration specified by the velocitySet command, then halt motion. During the move the enabled substate will be ConstantVelocity, then it will return to Stationary.</Description>
-    <item>
-        <EFDB_Name>state</EFDB_Name>
-        <Description>state</Description>
-        <IDL_Type>boolean</IDL_Type>
-        <Units>dimensionless</Units>
-        <Count>1</Count>
-    </item>
-</SALCommand>
-
-<SALCommand>
-    <Subsystem>Rotator</Subsystem>
-    <Version>3.5.1</Version>
-    <Author></Author>
-    <EFDB_Topic>Rotator_command_velocitySet</EFDB_Topic>
```

Comment:
same comment as above. I am OK with merging this and the above into one command.

Reply:
See above.